src/cap-description.adoc

```diff
@@ -489,6 +490,9 @@ if ( (E < (CAP_MAX_E - 1)) & (t[MXLEN: MXLEN - 1] - b[MXLEN - 1] > 1) )
 That is, invert the most significant bit of _t_ if the decoded length of the
 capability is larger than E.
 
+NOTE: A capability has infinite bounds if E=CAP_MAX_E and it is not
```

Comment:
Maybe it's worth being more explicit about the relationship between `t>=2^MXLEN` and `E=CAP_MAX_E`. Are they exactly equivalent? Something like:

```
NOTE: Checking for infinite bounds via `b= 0` and `t >= 2^MXLEN` is exactly equivalent to checking `E=CAP_MAX_E` and that the capability is not malformed.
```

(if that is the case)

Reply:
Yep!
`(inCap.E == cap_max_E & not(isCapMalformed(inCap))) == (base == 0 & top >= 2^(sizeof(xlen)))`

Reply:
Maybe a nice property to try and prove btw!

Reply:
Yes, sorry, should have been clearer: that's a property that SMT was happy with.

Reply:
Oh right, nice! Definitely worth pointing that out in the spec then I think.

Reply:
Fixed